src/uwtools/tests/test_schemas.py

```diff
@@ -1664,6 +1664,18 @@ def test_schema_rocoto_workflow_cycledef():
     assert "'foo' is not valid" in errors([{"attrs": {"activation_offset": "foo"}, "spec": spec}])
 
 
+def test_schena_rocoto_task_resources():
+    errors = schema_validator("rocoto", "$defs", "task", "properties")
+    # Basic resource options
+    assert not errors([{"cores": 1}])
+    assert not errors([{"native": "abc"}])
+    assert not errors([{"native": {"cyclestr": {"value": "def"}}}])
+    assert not errors([{"nodes": "1:ppn=12"}])
+    # Combined valid resources
+    assert not errors([{"cores": 1, "native": "abc"}])
+    assert not errors([{"nodes": "1:ppn=12", "native": "abc"}])
```

Comment:
I really wanted there to be an extra "don't allow cores and nodes" rule but do not have time to work out the logic with this bug fix. Maybe in a future PR?

Reply:
Do you mean that we'd like to enforce that `cores` and `nodes` are mutually exclusive? I think that can be done in the schema and tested here. If you open a Jira ticket for it, it'd be a nice fill-in task.

Reply:
I think that is how we should go, yes. @WeirAE Does your 👍 mean that you're creating a Jira ticket?